index.js

```diff
@@ -0,0 +1,95 @@
+const fs = require("fs");
+const { Readable } = require("node:stream");
+
+class APIClient
+{
+  static async login(url, username, password) {
+    const loginUrl = url + "/api/auth/jwt/login";
```

Comment:
Might be good to strip the trailing slash in `url` if provided, otherwise it'll cause 404s in a few of the methods.